app/src/main/res/layout/item_list_row.xml

```diff
@@ -1,28 +1,50 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/list_item"
     android:orientation="vertical"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_margin="10sp"
     android:onClick="showItemDescription">
+
     <RelativeLayout
-        android:orientation="vertical"
-        android:layout_width="match_parent"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_margin="10sp">
-        <TextView
-            android:id="@+id/list_item_name"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="name"/>
-        <TextView
-            android:id="@+id/list_item_price"
-            android:layout_width="wrap_content"
+        android:layout_margin="10dp"
+        android:orientation="horizontal">
+
+        <ImageView
```

Comment:
I like the the proportions you chose for the layout of the item card (image with the text) !